<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tcp_Client</class>
 <widget class="QMainWindow" name="tcp_Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tcp_Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>571</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>IP：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConnect">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>10</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>连接</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtIP">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPort">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>51</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>1234</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>端口：</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>571</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextEdit" name="txtReceive">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>551</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSend">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>410</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSend">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>361</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClear">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>410</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
